BankSDK/GiniBankSDK/Sources/GiniBankSDK/Resources/GiniColorScheme.swift

```diff
@@ -0,0 +1,256 @@
+        var divider: GiniColor
+    }
+
+    struct NavigationBar {
+        var bg: GiniColor
+        var action: GiniColor
+        var navigation: GiniColor
+    }
+
+    struct Content {
+        var navigation: GiniColor
+        var text: GiniColor
+    }
+
+    struct BottomBar {
+        var bg: GiniColor
+        var border: GiniColor
+    }
+    
+    struct Card {
+        var bg: GiniColor
+    }
+
+    struct Placeholder {
+        var bg: GiniColor
+        var tint: GiniColor
+    }
+
+    struct Text {
+        var primary: GiniColor
+        var secondary: GiniColor
+        var tertiary: GiniColor
+        var accent: GiniColor
+        var success: GiniColor
+        var error: GiniColor
+    }
+
+    struct Icons {
+        var primary: GiniColor
+        var secondary: GiniColor
+    }
+
+    struct PageController {
+        var selected: GiniColor
+        var unselected: GiniColor
+    }
+
+    struct Error {
+        var bg: GiniColor
+        var icon: GiniColor
+    }
+
+    struct Message {
+        var bgSuccess: GiniColor
+        var contentSuccess: GiniColor
+        var bgWarning: GiniColor
+        var contentWarning: GiniColor
+        var bgError: GiniColor
+        var contentError: GiniColor
+    }
+
+    struct Badge {
+        var bg: GiniColor
+        var content: GiniColor
+    }
+
+    struct Button {
+        var bg: GiniColor
+        var bgLoading: GiniColor
+        var content: GiniColor
+    }
+
+    struct ButtonOutlined {
+        var bg: GiniColor
+        var content: GiniColor
+    }
+
+    struct TextField {
+        var bg: GiniColor
+        var border: GiniColor
+        var focusedText: GiniColor
+        var unfocusedText: GiniColor
+        var disabledText: GiniColor
+        var errorText: GiniColor
+        var labelFocused: GiniColor
+        var labelUnfocused: GiniColor
+        var labelDisabled: GiniColor
+        var labelError: GiniColor
+        var supportingFocused: GiniColor
+        var supportingUnfocused: GiniColor
+        var supportingDisabled: GiniColor
+        var supportingError: GiniColor
+        var trailingFocused: GiniColor
+        var cursorEnabled: GiniColor
+        var cursorError: GiniColor
+    }
+
+    struct Toggles {
+        var thumb: GiniColor
+        var trackOn: GiniColor
+        var trackOff: GiniColor
+        var disabledTrack: GiniColor
+    }
+
+    var bg: Bg
+    var actionSheets: ActionSheets
+    var alert: Alert
+    var navigationBar: NavigationBar
+    var content: Content
+    var bottomBar: BottomBar
+    var card: Card
+    var placeholder: Placeholder
+    var text: Text
+    var icons: Icons
+    var pageController: PageController
+    var error: Error
+    var message: Message
+    var badge: Badge
+    var button: Button
+    var buttonOutlined: ButtonOutlined
+    var textField: TextField
+    var toggles: Toggles
+}
+
+extension UIColor {
+    public static func giniColorScheme() -> GiniColorScheme {
```

Comment:
[suggestion] To improve readability, maintainability, and avoid redundancy, you can refactor the giniColorScheme() method by breaking it into smaller, focused functions that initialize the different parts of the GiniColorScheme.
Like:
```
public static func giniColorScheme() -> GiniColorScheme {
    return GiniColorScheme(
        bg: createBgColorScheme(),
        actionSheets: createActionSheetsColorScheme(),
        alert: createAlertColorScheme(),
.....
    )
}

private static func createBgColorScheme() -> GiniColorScheme.Bg {
    return GiniColorScheme.Bg(
        primary: GiniColor(light: .GiniBank.light2, dark: .GiniBank.dark2),
        secondary: GiniColor(light: .GiniBank.light1, dark: .GiniBank.dark2)
    )
}
.....
```